File: src/ClaudeToolbar.Core/Formatting/PercentFormatter.cs
namespace ClaudeToolbar.Core.Formatting;

public static class PercentFormatter
{
    public static string Format(double utilization)
    {
        var clamped = Math.Clamp(utilization, 0, 100);
        var rounded = (int)Math.Round(clamped, MidpointRounding.AwayFromZero);
        return $"{rounded}%";
    }
}

File: mac/Sources/ClaudeToolbarCore/Formatting/PercentFormatter.swift
import Foundation

public enum PercentFormatter {
    public static func format(_ utilization: Double) -> String {
        let clamped = min(max(utilization, 0), 100)
        let rounded = Int(clamped.rounded(.toNearestOrAwayFromZero))
        return "\(rounded)%"
    }
}
